cli.go

```diff
@@ -516,3 +520,7 @@ func handleInterrupt(cancel context.CancelFunc) {
 	<-c
 	cancel()
 }
+
+func stripComments(input string) string {
```

Comment:
Just a random idea.

## Random idea 1

What do you think about changing `gsqlsep.InputStatement` as follows,

```
type InputStatement struct {
	Statement                string
	StatementWithoutComment  string
	Terminator               string
}
```

and use `StatementWithoutComment` for input match and `Statement` for other use case? Then we don't need to call `gsqlsep.Separate...` functions in two difference places.

## Random idea 2

What do you think about adding `StripComments()` method to `gsqlsep.InputStatement`?